jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/KubernetesHelperTest.java

```diff
@@ -488,8 +489,14 @@ void exportKubernetesClientConfigToFile_worksWithKubernetesMockServer(@TempDir P
             .singleElement(InstanceOfAssertFactories.type(NamedCluster.class))
             .hasFieldOrPropertyWithValue("cluster.insecureSkipTlsVerify", true)
             .extracting("cluster.server", "name")
-            .allMatch(s -> s.toString().matches("(https://)?localhost:\\d+/?"))
-          )
+                .satisfies(serverUrls -> {
+                  List<String> actualUrls = ((List<?>) serverUrls).stream()
+                      .map(Object::toString)
+                      .collect(Collectors.toList());
+                  assertThat(actualUrls).hasSize(2)
+                      .contains("https://" + mockServer.getHostName() + ":" + mockServer.getPort() + "/",
```

Comment:
Can we use `String.format("https://%s:%d/", mockServer.getHostName(), mockServer.getPort())` here?

Reply:
Umm, it was just my personal preference ( I find it more readable) . It's okay like this as well.